docs/manage/security/access-keys.md

```diff
@@ -36,22 +36,24 @@ Managing Access Keys (how to).
 
 ### From the Preferences page
 
-If you have the **Create Access Keys** role capability, you can use the **Preferences** page to create access keys.
+If you have the [**Create Access Keys** role capability](/docs/manage/users-roles/roles/role-capabilities#security), you can use the **Preferences** page to create access keys.
 
 To create your own access keys:
 
 1. In Sumo Logic, click your name in the left-nav and open the **Preferences** page.
-1. In the **My Access Keys** section, click **+ Add Access Key**.<br/>  ![access-key-preferences-page-2.png](/img/security/access-key-preferences-page-2.png)
-1. The **Create a Sumo Logic Access Key** window appears.<br/>  ![create-access-key.png](/img/security/create-access-key.png)
+1. In the **My Access Keys** section, click **+ Add Access Key**.<img src={useBaseUrl('/img/security/access-key-preferences-page-2.png')} alt="Add Access Key" width="1200"/>
```

Comment:
1200 px is massive for a repo image. For super wide UI screenshots like this one, try to keep them 600px or under.

Reply:
@kimsauce, thanks for this catch! I didn't realize that large screenshots slowed the repo builds. After looking at the wide UI screenshots at 600, I think they're too small, at least on my laptop. I think they look best at 800, so that's what I changed them to. Hope that's okay.